script/generate-release-notes.js

```diff
@@ -153,8 +161,11 @@ async function run() {
 
   for (const mergeCommitMessage of mergeCommitMessages) {
     const match = mergeCommitRegex.exec(mergeCommitMessage)
-    if (match.length === 2) {
-      pullRequestIds.push(match[1])
+    if (match != null && match.length === 2) {
```

Comment:
`tsc` caught a bug here where I wasn't handling `match` being `null` :tada:. There's an upcoming Octokit change here where it will expect a `number` later on, so I'm going to do that parsing here as a sanity check.